packages/alfa-xpath/src/syntax/lexer.ts

```diff
@@ -253,7 +254,8 @@ const lexCommentContents: Parser<Slice<number>, string> = (input) => {
           input = input.slice(1);
           value += ":";
 
-          const [remainder, result] = lexCommentContents(input).get();
+          // this function never returns Err
+          const [remainder, result] = lexCommentContents(input).getUnsafe();
```

Comment:
**Thought:** Should that be an infaillible parser, then?

Reply:
I ended up making it infallible and adding a `Parser.Infallible#toParser` function.